src/components/Form/Example.md

```diff
@@ -160,3 +160,33 @@ const { Form, Button, TextField } = require('precise-ui');
   </div>
 </Form>
 ```
+
+**Using Prompt with Form**
+
+When form has changes, it is possible to make a prompt, if user wants to leave the page without submitting the form. `Form` component has built-in `Prompt` component.
+
+Simple way to use it is to provide `prompt` prop. It will show browser's system dialog.
+
+```jsx
```

Comment:
There is no sense to make a snapshot of this example.

```suggestion
```jsx { "props": { "data-skip": true } }
```